packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -11,6 +11,7 @@ import 'dart:ui' show AppLifecycleState;
 import 'package:collection/collection.dart' show PriorityQueue, HeapPriorityQueue;
 import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
+import 'package:flutter/src/logging/logging.dart';
```

Comment:
it's not clear to me why this would be in the scheduler binding? what's it got to do with scheduling?

Reply:
It doesn't have to be.  I just wanted registration to happen early.

Reply:
Happy to move it to where you think is best!

Reply:
the scheduler might not be included at all, so putting it here isn't a good idea if you want it to happen early. :-)